src/angular/src/app/spark-core-angular/directives/inputs/sprk-selection-input/sprk-selection-input.directive.ts

```diff
@@ -0,0 +1,37 @@
+import { Directive, Input, OnInit, OnChanges, ElementRef } from '@angular/core';
+
+@Directive({
+  selector: '[sprk-selection-input]'
+})
+
+export class SprkSelectionInputDirective implements OnInit, OnChanges {
+  @Input() additionalClasses: string;
+  @Input() valid: boolean = true;
```

Comment:
I don't remember completely, but wasn't there some issue we ran into with booleans as inputs?

Reply:
nope, just have to surround the attribute with [] when using